### PR TITLE
Update ingress posthook to use acme.domains

### PR DESCRIPTION
**Pull request by baluchicken**

### CODE REVIEW

File: cluster/hooks.go [hunk 1]
```diff
@@ -541,6 +549,18 @@ func InstallIngressControllerPostHook(input interface{}) error {
 		"delayDontCheckDNS": 60,
 		"email":             user.Email,
 		"persistence":       map[string]interface{}{"enabled": true},
+		"domains": map[string]interface{}{
```
Comment (sagikazarmark) [inline cluster/hooks.go]: Can we create a struct for this whole thing?

File: cluster/hooks.go [hunk 2]
```diff
@@ -51,6 +51,11 @@ import (
 	pkgHelmRelease "k8s.io/helm/pkg/proto/hapi/release"
 )
 
+type domains struct {
+	Enabled     bool                     `json:"enabled"`
+	DomainsList []map[string]interface{} `json:"domainsList"`
```
Comment (stoader) [inline cluster/hooks.go]: typo, should be `domainList`

Comment (kozmagabor) [reply cluster/hooks.go]: Nope, It's ok.
https://github.com/helm/charts/tree/master/stable/traefik#lets-encrypt-wildcard-certificate

File: cluster/hooks.go [hunk 3]
```diff
@@ -533,6 +538,27 @@ func InstallIngressControllerPostHook(input interface{}) error {
...
+				"main": fmt.Sprintf("*.%s", domainWithOrgName)},
+			{
+				"sans": []string{
+					fmt.Sprintf("%s.%s", cluster.GetName(), domainWithOrgName),
+					domainWithOrgName},
+			},
+		},
+	}
+
 	acme := map[string]interface{}{
```
Comment (sagikazarmark) [inline cluster/hooks.go]: Sorry, I might not have been clear: the whole thing could be refactored into struct

**Pull request merged**